package.json

```
@@ -14,24 +15,23 @@
     "#dev-util": "./dev/src/util.js"
   },
   "scripts": {
+    "commit": "cz",
     "cp": "mkdir -p ./dev/public/shiki && cp -r ./node_modules/shiki/{dist,languages,samples,themes} ./dev/public/shiki",
     "cp:shikifix": "cp node_modules/.pnpm/vscode-oniguruma@1.7.0/node_modules/vscode-oniguruma/release/onig.wasm ./dev/public/shiki/dist/",
     "dev": "vite dev dev --config ./vite.config.js",
     "test": "vitest run",
     "watch": "vitest watch",
     "build": "rm -rf dist && rollup -c",
-    "publish:eik": "npx @eik/cli publish && npx @eik/cli pkg-alias",
+    "publish:eik": "eik publish && eik pkg-alias",
     "site": "vite build dev --config ./vite.config.js",
     "postsite": "cd dev/dist && cp index.html 200.html && cp index.html 404.html && cp ../CNAME .",
-    "preversion": "npm run test && npx @eik/cli login && npm run build",
+    "preversion": "pnpm run test && eik login && pnpm run build",
     "version": "npm publish",
-    "postversion": "npm run publish:eik && git push --follow-tags"
+    "postversion": "pnpm run publish:eik && git push --follow-tags"
```

Do we need those now? and preversion

```suggestion
    "postversion": "pnpm run publish:eik && git push --follow-tags"
```

---

Probably not? I kept them because I didn't know at this point. 😄  ...but perhaps we should just delete them, and re-introduce from Fabric if needed?

---

We do have another publishing procedure so this can be removed.